toc/working-groups/app-runtime-platform.md

```diff
@@ -101,8 +101,10 @@ Components from the Diego, Garden, HAproxy, Logging and Metrics, Networking, Win
   * https://github.com/cloudfoundry/clock
   * https://github.com/cloudfoundry/consuladapter
   * https://github.com/cloudfoundry/debugserver
+  * https://github.com/cloudfoundry/diego-ci
   * https://github.com/cloudfoundry/diego-dockerfiles
   * https://github.com/cloudfoundry/diego-logging-client
+  * https://github.com/cloudfoundry/diego-team
```

Comment:
This repo doesn't exist or is a private repo.

In my opinion, there should not be any private repos listed as technical assets of a WG.

Reply:
FWIW, there are other private repos already on this list (networking-oss-deployments for example)

Reply:
Discussion in TOC meeting: For now we can have a separate group of private repos under technical assets. This makes it clear to everyone that these repos are not explorable for the whole community.

Long term we should try to get rid of the private repos

Reply:
@geofffranks: Any update from your side? Can you put the private repos into a separate section?

Reply:
Sorry, I missed that comment. Will do!